tests/integration/test_ethernet.py

```diff
@@ -12,7 +12,11 @@
 except ImportError:
     import mock
 
-parentdir = os.path.normpath(os.path.join(os.path.dirname(__file__), "../../"))
+_is_role = os.path.exists(os.path.join(os.path.dirname(__file__), "../roles/linux-ystem-roles.network"))
```

Comment:
linux-system-roles.network

Reply:
*shame shame...* Now I wonder why my tests (especially, in the legacy role env) successfully ran...